From fdcf0b86c8993d584d237b33e083719b8936dbe4 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Fri, 26 Mar 2021 15:40:32 +0200
Subject: [PATCH 1/1] Apply Solus defaults.

These defaults are.

- Enabling tap-to-click by default
- Setting rgba antialiasing
- Setting menu:minimize,maximize,close window decorations
---
 data/org.gnome.settings-daemon.peripherals.gschema.xml.in     | 2 +-
 ...org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/org.gnome.settings-daemon.peripherals.gschema.xml.in b/data/org.gnome.settings-daemon.peripherals.gschema.xml.in
index be672773..cb8a7154 100644
--- a/data/org.gnome.settings-daemon.peripherals.gschema.xml.in
+++ b/data/org.gnome.settings-daemon.peripherals.gschema.xml.in
@@ -112,7 +112,7 @@
       <description>Select the touchpad scroll method. Supported values are: “disabled”, “edge-scrolling”, “two-finger-scrolling”.</description>
     </key>
     <key name="tap-to-click" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Enable mouse clicks with touchpad</summary>
       <description>Set this to TRUE to be able to send mouse clicks by tapping on the touchpad.</description>
     </key>
diff --git a/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in b/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in
index 45382f74..43ea08d4 100644
--- a/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in
+++ b/data/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.in
@@ -12,14 +12,14 @@
       <description>A list of strings representing the GTK+ modules that will be loaded, usually in addition to conditional and forcibly disabled ones.</description>
     </key>
     <key type="a{sv}" name="overrides">
-      <default>{}</default>
+      <default><![CDATA[{'Gtk/ShellShowsAppMenu': <0>, 'Gtk/DecorationLayout': <'menu:minimize,maximize,close'>}]]></default>
       <summary>A dictionary of XSETTINGS to override</summary>
       <description>This dictionary maps XSETTINGS names to overrides values. The values must be either strings, signed int32s or (in the case of colors), 4-tuples of uint16 (red, green, blue, alpha; 65535 is fully opaque).</description>
     </key>
   </schema>
   <schema id="org.gnome.settings-daemon.plugins.xsettings.deprecated">
     <key name="antialiasing" enum="org.gnome.settings-daemon.GsdFontAntialiasingMode">
-      <default>'grayscale'</default>
+      <default>'rgba'</default>
       <summary>Antialiasing</summary>
       <description>The type of antialiasing to use when rendering fonts. Possible values are: “none” for no antialiasing, “grayscale” for standard grayscale antialiasing, and “rgba” for subpixel antialiasing (LCD screens only).</description>
     </key>
-- 
2.30.2

